### PR TITLE
add some setter for pkg/mtls/crypto/tls.Conn

### PR DESCRIPTION
**Pull request by antJack**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

**Review by taoyuanyuan — DISMISSED** [review]
LGTM

File: pkg/mtls/crypto/tls/tls_custom.go [hunk 1]
```diff
@@ -201,6 +202,30 @@ func (c *Conn) GetConnectionState() gotls.ConnectionState {
 	return state
 }
 
+func (c *Conn) SetConnectionState(state gotls.ConnectionState) {
```
Comment (nejisama) [inline pkg/mtls/crypto/tls/tls_custom.go:205]: 这个改动是不是应该在readme里加一个记录

Comment (antJack) [reply pkg/mtls/crypto/tls/tls_custom.go:205]: done

**Review by taoyuanyuan — APPROVED** [review]

**Review by wangfakang — APPROVED** [review]
LGTM

**Pull request merged**